runtime/src/lib.rs

```diff
@@ -139,7 +139,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     //   `spec_version`, and `authoring_version` are the same between Wasm and native.
     // This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
     //   the compatible custom types.
-    spec_version: 194,
+    spec_version: 162,
```

Comment:
?

Reply:
I recommend this crate https://crates.io/crates/substrate-spec-version, helps grab the spec version quick

Reply:
you mean my crate :P